sdk/eventgrid/eventgrid/README.md

```diff
@@ -19,13 +19,14 @@ Use the client library to:
 ### Currently supported environments
 
 - Node.js version 8.x.x or higher
+- Browser JavaScript
```

Comment:
a browser list would make more sense here, since I don't think you want to claim IE support even though it *does* support JS. ;)